layouts/partials/head.html

```diff
@@ -89,10 +85,12 @@
 <meta property="og:type" content="{{ $ogType }}">
 
 {{ if .HasShortcode "mermaid" }}
-<!-- Copied from https://unpkg.com/mermaid@8.11.2/dist/mermaid.min.js -->
+<!-- Copied from https://unpkg.com/mermaid@8.13.4/dist/mermaid.min.js -->
 <script async src="{{ "js/mermaid.min.js" | relURL }}"></script>
 {{ end }}
```

Comment:
This PR upgraded `mermaid.js` from `v8.11.2` to `v8.13.4`. I think it's inappropriate to use the same filename.

Please revert this change or append the version number to the filename.

Reply:
I'd vote for appending the version number to the filename.

Reply:
NP, made the change. I would like @sftim opinion on need to make changes that do not impact problem fix.

Reply:
When making unrelated or semi-related changes, I like to see those in their own commit (makes code history searches easier, or even reverts if we need to do that).